### ISSUE TITLE
Consistent naming of block fields in the transaction list

### ISSUE THREAD

**Issue by davidyuk** [aeternity/ae_mdw#180]
can't reopen #112

Freezing the api and refusing to fix existing issues is not a good way to go. I'm proposing to collect a set of improvements like in mentioned issue and create a new set of endpoints (`/v2/...`) as the code team did in node. This can be introduced without breaking the current API allowing developers to gradually switch to a new one.

**Comment by marc0olo** [comment]
yes, we definitely shouldn't break the existing api! cc @mradkov

**Comment by thepiwo** [comment]
what are the specific requests here?

**Comment by davidyuk** [comment]
- try to find a better name for `micro_time` (is it a time in microseconds?)
- consider combining all block-related fields into a single object, i.e. to use nesting in json more actively
- ensure that `block_hash`, `block_height`, `micro_index`, `micro_time` documented somehow
